test/Runtime/Base.hs

```diff
@@ -24,6 +24,30 @@ clangCompile mkClangArgs inputFile outputFile execute step =
         execute outputFile'
     )
 
+readProcess :: FilePath -> [String] -> Text -> IO Text
```

Comment:
Explain briefly what this function does

Reply:
The same as `P.readProcess` but instead of inheriting `stderr` redirects it to the child's `stdout`. I'll add a comment.